web/components/answers/answers-panel.tsx

```diff
@@ -235,16 +251,36 @@ function OpenAnswer(props: {
           </Row>
           <Row className="gap-2">
             <div className="my-auto text-xl">{probPercent}</div>
-            {tradingAllowed(contract) && (
-              <Button
-                size="2xs"
-                color="gray-outline"
-                onClick={() => setOpen(true)}
-                className="my-auto"
-              >
-                BUY
-              </Button>
-            )}
+            {tradingAllowed(contract) &&
+              (isFreeResponse ? (
```

Comment:
You need to handle DPM multiple choice markets as well.

Reply:
ok, good catch